### PR TITLE
Add a function to ensure an integer is returned

### PR DESCRIPTION
**Pull request by olafura**
Sometimes we have to make sure we are getting integers even if
the source is a string so this function helps with that

### CODE REVIEW

**Review by enricribas — APPROVED** [review]

**Review by Corvillus — APPROVED** [review]

File: lib/kitchen_sink/integer.ex [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+  @doc """
+  Make sure that we get an integer either from an integer or string
+
+  iex> ensure_integer("1")
+  1
+  iex> ensure_integer(2)
+  2
+  """
+  def ensure_integer(i) when is_integer(i), do: i
```
Comment (mariusbutuc) [inline lib/kitchen_sink/integer.ex:14]: `Integer.ensure_integer/1` might be a bit much. 

Naming is hard; some raw thoughts would be naming the module `Convert` or `Cast` and the function `to_integer` or something along those lines?

Comment (olafura) [reply lib/kitchen_sink/integer.ex:14]: Then I would be tempted to use Protocols :D

Comment (olafura) [reply lib/kitchen_sink/integer.ex:14]: There is also https://github.com/DanielZendejas/lazy_integer

Comment (mariusbutuc) [reply lib/kitchen_sink/integer.ex:14]: Nice idea behind `LazyInteger`! 

At a quick glance @olafura, it seems to be Ecto-specific…

Comment (mariusbutuc) [reply lib/kitchen_sink/integer.ex:14]: @olafura on a second glance, we also have `Boolean`, `Map` etc. so `Integer` makes sense in this context.

`Integer.cast/1` could be an option too, but don't have a strong feeling about it. :shipit:

**Review by mariusbutuc — APPROVED** [review]

File: lib/kitchen_sink/integer.ex [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  def ensure_integer(s) when is_binary(s), do: String.to_integer(s)
```
Comment (maedhr) [inline lib/kitchen_sink/integer.ex]: Should we trim the string so it doesn't blow up?

**Review by maedhr — APPROVED** [review]

**Review by maedhr — CHANGES_REQUESTED** [review]
I meant to not approve last time... I will one-day learn to github.

**Comment by olafura** [comment]
:)

File: lib/kitchen_sink/integer.ex (hunk 2 above)
Comment (mariusbutuc) [reply lib/kitchen_sink/integer.ex]:
```elixir
def ensure_integer(s) when is_binary(s), do: s |> String.strip() |> String.to_integer()
```

**Review by maedhr — APPROVED** [review]

**Pull request merged**